### PR TITLE
Fix/separate manage controllers

### PR DESCRIPTION
**Pull request by stevenhillcox**

### CODE REVIEW

File: VotingApplication/VotingApplication.Web.Api.Tests/Controllers/ManageExpiryControllerTests.cs [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+                ManageExpiryController controller = CreateManageExpiryController(contextFactory);
+
+                controller.Put(PollManageGuid, request);
+
+                Assert.AreEqual(expiry, existingPoll.ExpiryDate);
+            }
+
+            [TestMethod]
+            [ExpectedHttpResponseException(HttpStatusCode.BadRequest)]
+            public void PostExpiry_IsRejected()
```
Comment (tpkelly) [inline VotingApplication/VotingApplication.Web.Api.Tests/Controllers/ManageExpiryControllerTests.cs]: Should this be Past not Post?

File: VotingApplication/VotingApplication.Web/Api/Controllers/ManagePollTypeController.cs [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                    List<Vote> removedVotes = context.Votes.Include(v => v.Poll)
+                                                            .Where(v => v.Poll.UUID == poll.UUID)
+                                                            .ToList();
+                    foreach (Vote oldVote in removedVotes)
+                    {
+                        context.Votes.Remove(oldVote);
+                    }
+
+                    poll.PollType = (PollType)Enum.Parse(typeof(PollType), updateRequest.PollType, true);
+                    poll.MaxPerVote = updateRequest.MaxPerVote;
```
Comment (tpkelly) [inline VotingApplication/VotingApplication.Web/Api/Controllers/ManagePollTypeController.cs]: The MaxPerVote should be editable if the poll type hasn't changed too

File: VotingApplication/VotingApplication.Web/VotingApplication.Web.csproj [hunk 3]
```diff
@@ -439,6 +443,7 @@
   </ItemGroup>
   <ItemGroup>
     <Folder Include="App_Data\" />
+    <Folder Include="Views\ManageExpiry\" />
```
Comment (tpkelly) [inline VotingApplication/VotingApplication.Web/VotingApplication.Web.csproj]: There doesn't appear to be anything in this folder

**Comment by tpkelly** [comment]
+1 Happy to merge if John is happy

**Comment by JDawes-ScottLogic** [comment]
:+1: :shipit:

**Pull request merged**